frameworks/Python/bottle/uwsgi.ini

```diff
@@ -2,7 +2,7 @@
 master
 ; Increase listen queue used for nginx connecting to uWSGI. This matches
 ; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
-listen = 65535
+listen = 128
```

Comment:
@methane it is 65535 for all setups. It is possible that it was set to 128 on one of the 3 machines during our preview round, but we will make sure it's correctly set for our next run of the suite.